app/services/atmosphere/nic_provider.rb

```diff
@@ -0,0 +1,7 @@
+module Atmosphere
+
+  class NicProvider
+    extend Atmosphere::NicProviderExt
```

Comment:
I use extend insted of include because I want to call NicProvider.nics instead of NicProvider.new.nics